extension/src/experiments/studio.ts

```diff
@@ -30,7 +33,11 @@ export class Studio extends Disposable {
       this.dvcRoot
     )
 
-    this.baseUrl = await this.getBaseUrl(studioToken, gitRemote)
+    this.baseUrl = await this.getBaseUrl(
+      studioToken,
+      gitRemote,
+      remoteExpRefs.length > 0 ? remoteExpRefs : [null]
```

Comment:
[F] Once there is a `GET` endpoint we won't call the `POST` endpoint if there are no new refs.